src/Models/Traits/UploadContentTrait.php

```diff
@@ -78,6 +79,11 @@ public function validateUploads(array $uploads, ?array &$errors = null): bool
             }
 
             foreach ($curUploads as $upload) {
+                // Skip validating this upload if already attached.
+                if ($attachedUploads->count() > 0 && $attachedUploads->where('id', $upload['id'])->isNotEmpty()) {
```

Comment:
We should be able to just check `content_id` on the `Upload` model rather than querying the attached uploads on the container